protocol-configuration/mainnetAddresses.ts

```diff
@@ -1676,6 +1676,12 @@ const MainnetAddresses: MainnetAddresses = {
     address: '0xFf419Bc27483edb94b7Ad5c97b7FaB5DB323c7E0',
     category: AddressCategory.Deprecated
   },
+  // TODO: Uncomment this FeiSkimmer and use once DAO vote has granted roles
```

Comment:
Commented until a DAO vote gives the FeiSkimmer the relevant roles (currently fails e2e tests)